spec/spec_helper.rb

```diff
@@ -1,8 +1,31 @@
-require 'rspec-puppet'
+require 'puppetlabs_spec_helper/module_spec_helper'
+require 'rspec-puppet-facts'
+include RspecPuppetFacts
+add_custom_fact :pe_version, '2016.4.2'
```

Comment:
As an example, we do the same here:
https://github.com/voxpupuli/puppet-collectd/blob/master/spec/default_module_facts.yml